arkouda/dataframe.py

```diff
@@ -2101,11 +3040,65 @@ def argsort(self, key, ascending=True):
         ----------
         key : str
             The key to sort on.
+        ascending : bool, default = True
+            If true, sort the key in ascending order.
+            Otherwise, sort the key in descending order.
 
         Returns
         -------
-        ak.pdarray
+        arkouda.pdarrayclass.pdarray
```

Comment:
Is there a specific reason we are changing these away from `ak.`?

Reply:
arkouda.pdarrayclass.pdarray or pdarray, sphinx will automatically create a clickable link to the class definition, whereas ak.pdarray will simply be shown as a string.  I think the links in the docs are helpful.

Reply:
Yup, that makes total sense to me. Thank you for clarifying